Bio/GenBank/__init__.py

```diff
@@ -1194,7 +1194,15 @@ def feature_qualifier(self, key, value):
             # otherwise just skip this key
             return
 
-        value = value.replace('"', '')
+        # remove enclosing quotation marks
+        value = re.sub('^"|"$', '', value)
+
+        # Handle escaping requirements for quotation marks
+        # remove escaping if exists
+        value = re.sub('""', '"', value)
```

Comment:
Why not write this as:

```python
# Undo escaping, repeated double quotes -> one double quote
value = value.replace('""', '"')
```

Reply:
Also, can you see a nice way to spot a double quote character which is not part of a two double quote pair? If so we can raise a parser warning about improperly escaped quotation marks.